### PR TITLE
Announce rotating JWKs

### PR DESCRIPTION
**Pull request by jalyna**
Screenshot 2022-04-25 at 14 38 46

### CODE REVIEW

File: docs/changelog/2022-05-09_new-jwks.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  If you are verifying Zaikio signed JSON Web Tokens (JWTs) as described in our [JWT Validation guide](https://docs.zaikio.com/guide/jwt/), please read this announcement. If you are not offering an API that accepts Zaikio's JWTs (e.g. you are consuming the Procurement API), you do not need to take any action.
+
+
+  In order to improve regular security measures we will start rotating keys that are used to sign Zaikio's JWTs.
+
+
+  ### What do I need to do?
+
+
+  1. Make sure you are not hardcoding Zaikio's public keys from `/api/v1/jwt_public_keys`. Instead serve those keys directly from our API or cache them for an hour maximum.
```
Comment (nickcampbell18) [inline docs/changelog/2022-05-09_new-jwks.md:21]:
```suggestion
  1. Make sure you are not hardcoding Zaikio's public keys from `/api/v1/jwt_public_keys`. Instead read those keys directly from our API or cache them for an hour maximum.
```

File: docs/changelog/2022-05-09_new-jwks.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+  2. Make sure that your JWT library accepts multiple keys or that you try **all** keys. Have a look in our [JWT Validation guide](https://docs.zaikio.com/guide/jwt/) for some implementation examples. We will always have some older keys, that have still JWTs that are signed with those (e.g. a personal access token is valid for 1 year), which is why you always have to support all keys, that you can find in `/api/v1/jwt_public_keys`. **Our new public key is already available but not used yet to sign JWTs.**
```
Comment (nickcampbell18) [inline docs/changelog/2022-05-09_new-jwks.md:23]:
```suggestion
  2. Make sure that your JWT library accepts multiple keys and that you try **all published keys**. Have a look in our [JWT Validation guide](https://docs.zaikio.com/guide/jwt/) for some implementation examples. We will always have some older keys, that have still JWTs that are signed with those (e.g. a personal access token is valid for 1 year), which is why you always have to support all keys, that you can find in `/api/v1/jwt_public_keys`. **Our new public key is already available but not used yet to sign JWTs.**
```

**Review by nickcampbell18 — APPROVED** [review]
Looks good and I think the timeline is fair.

**Comment by cweyer** [comment]
Maybe we should emphasise that from now on we rotate keys at random? So people understand why they need to automate this.

**Review by cweyer — APPROVED** [review]

**Comment by jalyna** [comment]
@cweyer 

Added

> We will do future rotations unannounced and randomly, so please make sure you are following our instructions.

**Pull request merged**